config/routes.rb

```diff
@@ -241,7 +241,7 @@
 
   # You can have the root of your site routed with "root
   get "/about" => "pages#about"
-  get "/api", to: "pages#api"
+  get "/api", to: redirect("https://docs.dev.to/api")
```

Comment:
the existing "colloquial" API page has been replaced by the auto generated doc (which supports markdown in the descriptions by the way)